classes/output/slide_table.php

```diff
@@ -157,12 +157,21 @@ public function populate_block_table($blockid) {
             }
 
             // Setup the drag handles for DnD.
-            $actions .= \html_writer::span($OUTPUT->render_from_template('core/drag_handle',
-            ['movetitle' => get_string('move')]), '', [
-                'data-action' => 'move',
-                'data-rowid' => $id,
-                'data-name' => "Slide {$slidenum}"
-            ]);
+            try {
+                $actions .= \html_writer::span($OUTPUT->render_from_template('core/drag_handle',
+                    ['movetitle' => get_string('move')]), '', [
+                    'data-action' => 'move',
+                    'data-rowid' => $id,
+                    'data-name' => "Slide {$slidenum}"
+                ]);
+            } catch (\moodle_exception $e) {
```

Comment:
This would work fine, but I think the better pattern is an `if file_exists('draghandle.js')` and switch on that to decide just the string for  the JS name, that way we dont dupe the code block of params